packages/venia-concept/src/components/Checkout/wrapper.js

```diff
@@ -16,53 +15,51 @@ import {
 
 import Flow from './flow';
 
-const { BrowserPersistence } = Util;
-const storage = new BrowserPersistence();
-
-const isAddressValid = address => !!(address && address.email);
+const hasData = value => !!value;
 const isCartReady = cart => cart.details.items_count > 0;
 const isCheckoutReady = checkout => {
-    return (
-        isPaymentMethodReady() &&
-        isShippingInfoReady() &&
-        isShippingMethodReady(checkout)
+    const {
+        billingAddress,
+        paymentData,
+        shippingAddress,
+        shippingMethod
+    } = checkout;
+
+    return [billingAddress, paymentData, shippingAddress, shippingMethod].every(
+        hasData
```

Comment:
You can just use `.every(Boolean)` here.

Reply:
I do like the simplicity of `.every(Boolean)` but `.every(hasData)` makes it explicit what we're testing for and avoids some mental overhead for the reader.

I'm fine with either, really, but unless you feel strongly I'd like to keep this as is.